src/apps/media/src/app/components/FileModal/FileTypePreview.tsx

```diff
@@ -34,6 +40,8 @@ export const FileTypePreview: FC<Props> = ({
   imageSettings,
 }) => {
   const theme = useTheme();
+  const is1440 = useMediaQuery("(min-width:1440px)");
```

Comment:
I'll check with Zosh if he's fine with MUI's default breakpoints since the one's I've used here were particularly provided by him. Also, do you think it's fine to change MUI's breakpoints to the values here? I was considering that but it might affect other components relying on the breakpoints.

Reply:
Updated to use theme breakpoints